cmd/magneticod/main.go

```diff
@@ -140,7 +141,8 @@ func parseFlags() (*opFlags, error) {
 		IndexerInterval     uint     `long:"indexer-interval" description:"Indexing interval in integer seconds." default:"1"`
 		IndexerMaxNeighbors uint     `long:"indexer-max-neighbors" description:"Maximum number of neighbors of an indexer." default:"10000"`
 
-		LeechMaxN uint `long:"leech-max-n" description:"Maximum number of leeches." default:"200"`
+		LeechMaxN   uint `long:"leech-max-n" description:"Maximum number of leeches." default:"200"`
+		MaxThrottle uint `long:"max-throttle" description:"Maximum requests per second." default:"0"`
```

Comment:
I would probably called this variable more precisely. Probably `MaxRps` and `--max-rps` or something more self-describing as a parameter name.